Userland/Utilities/nohup.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/LexicalPath.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+namespace {
+
+void dup_out_file(int fd_to_redirect)
+{
+    int fd = -1;
+    DeprecatedString path = "nohup.out";
+    auto options = O_CREAT | O_WRONLY | O_APPEND;
+    auto mode = S_IRUSR | S_IWUSR;
+    auto fd_or_error = Core::System::open(path, options, mode);
+
+    if (fd_or_error.is_error()) {
+        auto* home_env = getenv("HOME");
+        if (!home_env) {
+            warnln("nohup: unable to open for appending as $HOME is not set");
+            exit(127);
+        }
+
+        path = LexicalPath::join(LexicalPath::canonicalized_path(home_env), path).string();
+        fd_or_error = Core::System::open(path, options, mode);
+        if (fd_or_error.is_error()) {
+            warnln("nohup: unable to open for appending given path {}", path);
+            exit(127);
+        }
+    }
+
+    fd = fd_or_error.value();
+
+    if (dup2(fd, fd_to_redirect) == -1) {
+        dbgln("nohup: redirection failed: {}", strerror(errno));
+        exit(127);
+    }
+
+    if (fd_to_redirect != STDERR_FILENO)
+        out(stderr, "appending output to {}\n", path);
+}
+
+}
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    TRY(Core::System::pledge("stdio wpath cpath exec sigaction"));
+
+    DeprecatedString utility;
```

Comment:
If the LibC functions below can be replaced with their LibCore equivalents, then `StringView` can be used here and for `args` below.